### DISCUSSION TITLE
In map view, the ability to edit the distance that clustering groups records by

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13300] [Feature Requests]
Being able to cluster is great, but I'd like to be able to edit the distance that it groups data points by, so I thinking maybe a little slider in the ui to affect the grouping distance that's visible when cluster is enabled?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
